src/main/java/com/google/cloud/resourcemanager/spi/v1beta1/ResourceManagerRpc.java

```diff
@@ -133,4 +158,34 @@ Integer getInt(Map<Option, ?> options) {
    */
   Map<String, Boolean> testOrgPermissions(String resource, List<String> permissions)
       throws IOException;
+
+  // TODO(ajaykannan): implement "Organization" functionality when available (issue #319)
```

Comment:
don't need todo comment; the issue is enough

Reply:
done